src/vcpkg/binarycaching.cpp

```diff
@@ -952,68 +952,71 @@ namespace
     struct GHABinaryProvider : IBinaryProvider
     {
         GHABinaryProvider(const VcpkgPaths& paths,
-                          bool read,
-                          bool write,
-                          const Optional<std::string>& url,
-                          const Optional<std::string>& token)
+                          const bool read,
+                          const bool write,
```

Comment:
As a point of style, we usually do not mark function arguments as `const` -- especially in a short function like this constructor.

It's not wrong, it's just unusual.